cfme/tests/cloud/test_providers.py

```diff
@@ -597,11 +598,38 @@ def test_refresh_with_empty_iot_hub_azure(request, provider, setup_provider):
     assert result.validate(wait="60s")
 
 
+@pytest.fixture()
+def _azure_available_regions(provider):
```

Comment:
Since this not named within a class or function as a purely internal method, I would remove the preceding `_`.

```suggestion
def azure_available_regions(provider):
```

Same for the other two fixtures.